src/pybind/mgr/dashboard/frontend/src/app/shared/enum/health-label.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum HealthLabel {
+  HEALTH_ERR = 'ERROR',
+  HEALTH_WARN = 'WARNING',
+  HEALTH_OK = 'OK'
```

Comment:
Couple of things, what about making these strings [translatable](https://angular.io/api/localize)?
```suggestion
  HEALTH_ERR = $localize`ERROR`,
  HEALTH_WARN = '$localize`WARNING`,
  HEALTH_OK = $localize`OK`
```
And second (and somehow connected to internationalization): the all-uppercase is more of a style-presentation choice, right? ([not all languages have "capital" letters](https://www.quora.com/Which-languages-have-no-capitalized-letter)). So you could keep them in (default) lowercase and [only capitalize them](https://angular.io/api/common/UpperCasePipe) in the presentation stage:
```html
<div>{{ healthData.health.status | healthLabel | uppercase}}</div>
```

Reply:
@epuertat  It seems we can't use $localize directly in enums. 

![image](https://user-images.githubusercontent.com/15856785/179165977-182031a4-ceff-440a-bc85-34153902bdca.png)

I get the following error: 
"Only numeric enums can have computed members, but this expression has type 'string'. If you do not need exhaustiveness checks, consider using an object literal instead."

I found these two approaches:  [one](https://stackoverflow.com/questions/70390679/translate-enumerations-with-angular-i18n-native-module), [two](https://dev.to/constjs/angular-translate-enums-i18n-57k3?utm_source=dormosheio&utm_campaign=dormosheio)  but they seem too complex

Reply:
Solution 2 from the 2nd link (Using ICU) looks better to me, and we already use ICU expressions for singular/plurals in other places. That said it seems that we're [not the only ones looking for such support in TS](https://github.com/microsoft/TypeScript/issues/40793).

Reply:
Hi @epuertat, I am finding it difficult to implement the translations using ICU. Can you please share an example from the dashboard

Reply:
Ok, don't worry @nSedrickm . I think it's acceptable to leave the OK/WARNING/ERROR string untranslated (the former HEALTH_ERR, _WARN. and _OK were not translated either).